09-redux/classworks/Volodkova_Anna/src/App.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useState, useRef} from 'react';
+import {useRouteMatch} from "react-router-dom";
+import {useSelector, useDispatch} from 'react-redux';
+import Header from 'components/header'
+import Pages from 'pages';
+import {withRedditApi} from './hooks/useRedditApi';
+import {setPostsNull, setSearchValue, fetchPosts} from "./resourses/actions";
+
+function App(props) {
+  const dispatch = useDispatch();
+  const getSearchValue = useSelector((state) => state.value);
+  let getSubreddits = useSelector((state)=>state.posts);
```

Comment:
operator such as `+`, `-`, `*`, `/`, `%`, `&&`, `=>` and others should be separated by spaces from both sides